app/api/groups/add_child.go

```diff
@@ -20,8 +20,8 @@ import (
 //   Restrictions (otherwise the 'forbidden' error is returned):
 //     * the authenticated user should be an owner of both `parent_group_id` and `child_group_id,
 //     * the authenticated user should have `users.allowSubgroups` set to 1,
-//     * the parent group should not be of type "UserSelf",
-//     * the child group should not be of types "Base" or "UserAdmin",
+//     * the parent group should not be of type "UserSelf" or "Team",
+//     * the child group should not be of types "Base", "UserSelf", or "UserAdmin",
```

Comment:
We should add a note that users are not allowed as child as they need to use the createInvite/request services for that. Otherwise, we could wonder why "UserSelf" is not allowed here (the others are invalid relations while here it is just because it is handled differently)

Reply:
done